sample_metadata/parser/generic_parser.py

```diff
@@ -67,7 +67,7 @@
     + VCF_EXTENSIONS
 )
 
-rmatch = re.compile(r'[_\.-][Rr]?[12]')
+rmatch = re.compile(r'[_\.-][Rr]?[12]\.fastq(\.gz)?$')
```

Comment:
Should we care about `.fq.gz`?

Reply:
Addressed in latest commit @vladsaveliev, could you please confirm?

Reply:
Awesome, thank you!